Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -686,6 +686,23 @@ async def execute():
 
         self.assertIsNone(self.loop.run_until_complete(execute()))
 
+    def test_subprocess_communicate_stdout(self):
+        # See https://github.com/python/cpython/issues/100133
+        async def get_command_stdout(cmd, *args):
+            proc = await asyncio.create_subprocess_exec(
+                cmd, *args, stdout=asyncio.subprocess.PIPE,
+            )
+            stdout, _ = await proc.communicate()
+            return stdout.decode().strip()
+
+        async def main():
+            outputs = [f'foo{i}' for i in range(10)]
```

Comment:
Do we need 10? The repro from the issue had three repetitions and it always diagnosed the problem. Or is there a timing issue here where if the subprocesses are too fast the test may flake-succeed?

Reply:
Sometimes it passes even with 3 repetitions so I am overestimating it to 10.